encoding/thrift/thriftrw-plugin-yarpc/exception.go

```diff
@@ -42,11 +42,12 @@ package <$pkgname>
 
 <range $key, $val := .Types>
 	<if (isException $val)>
-	<$yarpcerrors := import "go.uber.org/yarpc/yarpcerrors" ->	
-	// Code is the <$yarpcerrors>.Code that should be associated with <$val.Name>.	
+	<$yarpcerrors := import "go.uber.org/yarpc/yarpcerrors" ->
+	// YARPCErrorCode returns <if isSetYARPCCode .Annotations>a <getYARPCErrorCode .><else> nil<end> for <$val.Name>.
+	//
```

Comment:
Do we really want this extra comment line?

Reply:
Yeah this was intentional. Consistent across the code base, we've tended to have the most important line as the first comment and noted supplementary / less important information as a new line.